examples/data/e2e_tests/test_jsons/castingConformanceRuleTest.json

```diff
@@ -0,0 +1,63 @@
+
+{
+    "vars": {
+      "spark-submit": "spark-submit --num-executors 2 --executor-memory 2G --deploy-mode client",
+      "spark-conf": "--conf 'spark.driver.extraJavaOptions=-Dmenas.rest.uri=http://localhost:8080/menas_war_exploded/api -Dspline.mongodb.name=spline -Dspline.mongodb.url=mongodb://127.0.0.1:27017/'",
```

Comment:
The e2e tests elsewhere refer to Menas running at `http://localhost:8080/menas`, perhaps unifying it here (and the related 11 occurrences) could lower the reader's confusion?